content/community/contributor-how-tos.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Contributor How-Tos
+description: ''
+---
+
+## About
+
+This document is intended to describe common and not-so-common processes that the contributor team executes as part of maintaining the 300+ open source projects within the Cloud Posse Organization.
+
+
+## How-Tos
+
+### Update Many Repos
+
+To update many of the open source repos with a common change such as updating Terraform `required_version` pinning, adding GitHub actions, or updating pinned providers, the contributor team has adopted using [microplane](https://github.com/Clever/microplane). This tool allows us to execute automated changes across dozens or even hundreds of our open source repos, which saves many man hours of contributor time.
```

Comment:
```suggestion
To update many of the open source repos with a common change such as updating Terraform `required_version` pinning, adding GitHub actions, or updating pinned providers, the contributor team has adopted using [microplane](https://github.com/Clever/microplane). This tool allows us to execute automated changes across dozens or even hundreds of our open source repos, which saves many hours of contributor time.
```

nitpick: More inclusive by omitting the word `man`

Reply:
Done.